src/services/account-states/config.ts

```diff
@@ -118,6 +118,6 @@ export const transitionConfiguration: TransitionConfigurationInterface = {
     AccountIsSuspended: ['02', '03', '04', '05', '06'],
     AccountNeedsPasswordReset: ['01', '02', '03', '05', '06', '25', '90', '94'],
     AccountNeedsIdReset: ['01', '02', '03', '04', '06', '91'],
-    AccountNeedsPswAndIdReset: ['01', '02', '03', '04', '05', '92', '93', '95'],
+    AccountNeedsPswAndIdReset: ['01', '02', '03', '04', '05', '25', '92', '93', '95'],
```

Comment:
I _think_ the '25' shouldn't be in the `AccountNeedsPswAndIdReset`? Or are we going to fix that with the tweaks I've done on my branch?